### PR TITLE
DB-9781: create missing index for system table

### PR DESCRIPTION
**Pull request by jyuanca**

### CODE REVIEW

**Comment by jyuanca** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/964/

**Comment by jyuanca** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: hbase_sql/src/main/java/com/splicemachine/hbase/HBaseRegionLoads.java [hunk 1]
```diff
@@ -234,6 +234,11 @@ private static String tableForRegion(String regionName){
     public Collection<PartitionLoad> tableLoad(String tableName, boolean refresh){
         if (refresh) {
             Map<String, Map<String, PartitionLoad>> loads = cache.get();
+            if (loads == null) {
```
Comment (dgomezferro) [inline hbase_sql/src/main/java/com/splicemachine/hbase/HBaseRegionLoads.java:237]: Is this related to this PR?

File: hbase_sql/src/test/java/com/splicemachine/hbase/CheckTableIT.java [hunk 2]
```diff
@@ -186,6 +192,72 @@ public static void dropTables() throws Exception {
...
+        rs = spliceClassWatcher.executeQuery(format(select, String.format("%s/fix-conglomerates.out", getResourceDirectory())));
+        String s = TestUtils.FormattedResult.ResultFactory.toStringUnsorted(rs);
+        rs.close();
+
+        rs = spliceClassWatcher.executeQuery("select rowid from sys.sysconglomerates --splice-properties index=null\n" +
+                "where conglomeratename='GI'");
+        rs.next();
+        rowid = rs.getString(1);
+
+        String expected = String.format("message                                   |\n" +
```
Comment (dgomezferro) [inline hbase_sql/src/test/java/com/splicemachine/hbase/CheckTableIT.java:245]: This is not used

File: splice_machine/src/main/java/com/splicemachine/derby/impl/storage/CheckTableJob.java [hunk 3]
```diff
@@ -100,17 +99,21 @@ public Void call() throws Exception {
...
         DataSetProcessor dsp = null;
         if (distributed) {
             SpliceLogUtils.info(LOG, "Run check_table on spark");
             dsp = EngineDriver.driver().processorFactory().distributedProcessor();
-        }
-        else {
+        } else {
             SpliceLogUtils.info(LOG, "Run check_table on region server");
             dsp = EngineDriver.driver().processorFactory().localProcessor(null, null);
         }
```
Comment (dgomezferro) [inline splice_machine/src/main/java/com/splicemachine/derby/impl/storage/CheckTableJob.java:119]: This doesn't make much sense to me, CheckTableJob is already running in the OlapServer, as if it was going to be running in Spark. But here we are choosing whether to run it in Spark or the RS. If we choose RS then as far as I can see it's going to run on the OlapServer instead. That decision should have been made in SpliceAdmin.checkIndexesInDetail() when submitting the checkTableJob to the OlapServer

File: hbase_sql/src/main/java/com/splicemachine/hbase/HBaseRegionLoads.java (hunk 1 above)
Comment (jyuanca) [reply hbase_sql/src/main/java/com/splicemachine/hbase/HBaseRegionLoads.java:237]: Somewhat. Erin hit this problem when she tried to fix a table

File: splice_machine/src/main/java/com/splicemachine/derby/impl/storage/CheckTableJob.java (hunk 3 above)
Comment (jyuanca) [reply splice_machine/src/main/java/com/splicemachine/derby/impl/storage/CheckTableJob.java:119]: This is not the problem I'm addressing in this PR. I'll file another Jira.

File: hbase_sql/src/test/java/com/splicemachine/hbase/CheckTableIT.java (hunk 2 above)
Comment (jyuanca) [reply hbase_sql/src/test/java/com/splicemachine/hbase/CheckTableIT.java:245]: good catch. I forgot to check results

**Comment by jyuanca** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1
